recipes-graphics/westeros/westeros-soc-drm.bb

```diff
@@ -13,7 +13,8 @@ DEPENDS = "wayland virtual/egl glib-2.0 libdrm"
 PROVIDES = "westeros-soc"
 RPROVIDES_${PN} = "westeros-soc"
 
-CFLAGS_append = " -I${STAGING_INCDIR}/libdrm"
+CFLAGS_append = " -I${STAGING_INCDIR}/libdrm -DWESTEROS_GL_NO_PLANES"
+CFLAGS_remove_rpi = "${@bb.utils.contains('MACHINE_FEATURES', 'vc4graphics', '-DWESTEROS_GL_NO_PLANES', '', d)}"
```

Comment:
LGTM. Requesting @wouterlucas  @kraj to review. 
FYI @kalyan-nagabhirava

Reply:
Will this need to be removed when using userland graphics driver ?

Reply:
No @kraj, this soc won't be applicable for userland (no drm) case.